README.md

```diff
@@ -117,326 +68,113 @@ https://pmi.pay1.de/
+`TENANT1_PAYONE_MODE`                 | Yes | the mode of operation with PAYONE <ul><li>`"live"` for production mode, (i.e. _actual payments_) or</li><li>`"test"` for test mode</li></ul> | `"test"`  
+`TENANT1_CT_START_FROM_SCRATCH`       | Yes | **WARNING** _**Handle with care!**_ If and only if equal, ignoring case, to `"true"` the service will create the custom types it needs. _**Therefor it first deletes all Order, Cart, Payment and Type entities**_. If not yet in the project, the Custom Types are created independently of this parameter (but only deleted and recreated if this parameter is set).  Related: [issue #34](https://github.com/commercetools/commercetools-payone-integration/issues/34). | `"false"`
+`TENANT1_SECURE_KEY`                  | Yes | if provided and not empty, the value is used as the key for decrypting data from fields "IBAN" and "BIC" for payments with CustomType "PAYMENT_BANK_TRANSFER". The data must be the result of a Blowfish ECB encryption with said key and encoded in HEX. | "" (empty String)
+`TENANT1_UPDATE_ORDER_PAYMENT_STATE`  | Yes | if _true_ - `Order#paymentState` will be updated when payment status notification is received from Payone. By default the order's state remains unchanged. | "false" 
 
 #### Docker run
 
 ```
 docker run \
--e CT_CLIENT_ID=xxx \
--e CT_CLIENT_SECRET=xxx \
--e CT_PROJECT_KEY=xxx \
--e PAYONE_KEY=xxx \
--e PAYONE_MERCHANT_ID=xxx \
--e PAYONE_MODE=test|live \
--e PAYONE_PORTAL_ID=xxx \
--e PAYONE_SUBACC_ID=xxx \
-sphereio/commercetools-payone-integration 
-```
-
-The integration service itself does not provide SSL connectivity, this must be done by a load balancer / SSL terminator running in front of it (which is recommended in any case).
- 
-#### Local run for development and tests
-
-  In the [`/build.gradle`](/build.gradle) script the [_application_](https://docs.gradle.org/current/userguide/application_plugin.html)
-  plugin is applied 
-  thus `gradle run` task could be used for local run and debug 
-  (for example, in Intellij IDEA you could _Run/Debug_ the gradle task `run` from the tasks list).
+    -e TENANTS=TENANT1 
+    -e TENANT1_CT_CLIENT_ID=xxx \
+    -e TENANT1_CT_CLIENT_SECRET=xxx \
+    -e TENANT1_CT_PROJECT_KEY=xxx \
+    -e TENANT1_PAYONE_KEY=xxx \
+    -e TENANT1_PAYONE_MERCHANT_ID=xxx \
+    -e TENANT1_PAYONE_MODE=test|live \
+    -e TENANT1_PAYONE_PORTAL_ID=xxx \
+    -e TENANT1_PAYONE_SUBACC_ID=xxx \
+sphereio/commercetools-payone-integration
+```
+
+## Multitenancy
+
+Starting from version 2 a single service instance may be used for multiple tenants, e.g. same service for different shop,
+merchants or suppliers. The next configuration properties are required for this:
+
+  * set mandatory `TENANTS` property as described in _[Mandatory common properties](#mandatory-common-properties)_ section above.
+  * setup _commercetools_ and _Payone_ properties for each specific tenant name, for example, if one has `TENANTS=BOOTS, BIKES`,
```

Comment:
based on my experience i'd allow to do this to keep readability. This, actually, was the main purpose of this - to allow set it up with commas, so the tenants names are easily recognized